homeassistant/components/sensor/__init__.py

```diff
@@ -95,6 +96,14 @@ async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     return await component.async_unload_entry(entry)
 
 
+@dataclass
+class SensorEntityDescription(EntityDescription):
+    """An class that describes sensor entities."""
+
+    state_class: str | None = None
+    last_reset: datetime | None = None
```

Comment:
Similarly, I've also not yet seen `last_reset` yet.

Reply:
last_reset is new and actively used by quite a few integrations. We hope this will rise more soon.

Reply:
Would not ResetType be more useful?

https://github.com/home-assistant/core/blob/9b2d98f0277f31f71bdd50e27f91a916870f41a7/homeassistant/components/tibber/sensor.py#L53-L58

Reply:
I think that is a different discussion. Unrelated to this PR.